README.md

```diff
@@ -149,43 +149,36 @@ You can add the Manufacture-Data by passing an Object of the Type __Organization
 __Example (groovy):__
 ```groovy
 cyclonedxBom {
-    //declaration of the Object from OrganizationalEntity
-    OrganizationalEntity organizationalEntity_temp = new OrganizationalEntity();
-
-    //setting the Name[String] and Url[List] of the Object
-    organizationalEntity_temp.setName("Test");
-    organizationalEntity_temp.setUrls(["www.test1.com", "www.test2.com"]);
     //declaration of the Object from OrganizationalContact
     OrganizationalContact organizationalContact = new OrganizationalContact();
     //setting the Name[String], Email[String] and Phone[String] of the Object
     organizationalContact.setName("Max_Mustermann");
     organizationalContact.setEmail("max.mustermann@test.org");
     organizationalContact.setPhone("0000 99999999");
-    //adding the Object[OrganizationalContact] to the Object[OrganizationalEntity]
-    organizationalEntity_temp.addContact(organizationalContact);
-
-    //passing organizationalEntity_temp to the plugin
-    organizationalEntity = organizationalEntity_temp;
+    //passing Data to the plugin
+    organizationalEntity {oe->
+        oe.name = 'Test'
+        oe.url = ['www.test1.com', 'www.test2.com']
+        oe.addContact(organizationalContact)
```

Comment:
I found no way to include the setting of OrganizationalContact into DSL

Reply:
Yes, I have had a look into it and it's not possible.